concepts/ranges/about.md

```diff
@@ -0,0 +1,125 @@
+
+Ranges can also be created using the `Range` initializer.
+
+```ruby
+Range.new(1, 5) # => 1, 2, 3, 4, 5
+```
+
+````exercism/note
+When creating a range in Ruby using the range operators `..` or `...`, and wanting to call a method on the range, you need to wrap the range in parentheses.
+This is because the otherwise will the method be called on the 2nd argument of the range operator.
+
+```ruby
+(1..5).sum # => 15
+1..5.sum # => Error: undefined method `sum' for 5:Integer (NoMethodError)
+```
+````
+
+## Getting substrings
+
+When wanting to slice a string, you can use the range operator to get a substring.
+That is by creating a range with the start and end index of the substring.
+
+```ruby
+"Hello World"[0..4] # => "Hello"
+"Hello World"[6..10] # => "World"
+```
+
+You can also use negative indexes to get the substring from the end of the string.
+
+```ruby
+"Hello World"[-5..-1] # => "World"
+"Hello World"[6..-4] # => "Wor"
+```
+
+## Range methods
+
+Ranges do have a set of methods that can be used to work with them.
+These methods when for example want to get the sum of all the values in the range, or check if the range includes a value.
+
+| Method                  | Description                                                             | Example                         |
+| ----------------------- | ----------------------------------------------------------------------- | ------------------------------- |
+| [`sum`][sum]            | Returns the sum of all the values in the range                          | `(1..5).sum # => 15`            |
+| [`size`][size]          | Returns the size of the range                                           | `(1..5).size # => 5`            |
+| [`include?`][indlude] | Returns `true` if the range includes the given value, otherwise `false` | `(1..5).include?(3) # => true` |
+
+## Endless & Beginless ranges
+
+There are two special types of ranges, the endless and beginless ranges.
+This means that the range has no beginning or end.
+The endless or beginless range has there start or end value being `nil`, but when defining the range so can nil be omitted.
+
+Using beginless and endless ranges is useful when you want to for example slice a string from the beginning or to the end.
+
+```ruby
+"Hello World"[0..] # => "Hello World"
+"Hello World"[4..] # => "o World"
+"Hello World"[..5] # => "Hello"
+```
+
+```exercism/caution
+If not used on a collection, the endless range can cause an infinite loop, if not used with caution.
+```
+
+## String ranges
+
+Strings can also be used in ranges and allow one to get an interval of strings between two strings.
+But its behavior is a bit different than with chars, when using multiple characters in a string range.
+Its behavior can become when doing more complex string ranges, so use it with caution.
+
+```ruby
+"aa".."az" # => "aa", "ab", "ac", ..., "az"
```

Comment:
```suggestion
In IRB:

>> "aa".."az"
=> "aa", "ab", "ac", # continues until  "az"
```

We could use a smaller sequence as well, and still demonstrate this, so we do not have to allude to the missing content.